DQM/CSCMonitorModule/plugins/CSCDQM_Collection.cc

```diff
@@ -156,8 +156,7 @@ namespace cscdqm {
       const XMLCh *content = element->getTextContent();
       XERCES_CPP_NAMESPACE_QUALIFIER TranscodeToStr tc(content, "UTF-8");
       std::istringstream buffer((const char*)tc.str());
-      std::string value;
-      buffer >> value;
+      std::string value = buffer.str();
```

Comment:
question: why?

Or why not just do `std::string value((const char*)tc.str());` and skip buffer entirely?

Reply:
Just minimal fix to have working code again, until I figure out how it was actually producing expected output before (which is a real question for me).

> Or why not just do std::string value((const char*)tc.str()); and skip buffer entirely?
Should work as well in this case

Reply:
@dmitrijus Now I think I figured it out.
It looks like the following commit on May 27, 2016 from central, related to UTF-8 handling, introduced this issue.
https://github.com/cms-sw/cmssw/commit/75ebcf5b556f44f3540ef74e14cdf6bea1045879#diff-1d119251269edd210e26561f4a3b2dbd